pkg/controller/manage/manage_controller.go

```diff
@@ -68,6 +68,22 @@ type KmeshManageController struct {
 	client            kubernetes.Interface
 }
 
+type podEventArgs struct {
+	namespaceLister v1.NamespaceLister
+	queue           workqueue.RateLimitingInterface
+	security        *kmeshsecurity.SecretManager
```

Comment:
"security" is a weird name, just use `secretManager` or use the abbreviation "sm".